recipes/flac/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "flac" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "e0c23c92af0aac830683a6ab6784dd0c11aaf04249709b9d600675b3b2441c3e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/xiph/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+    - cmake  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - pkgconfig  # [unix]
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - gettext  # [unix]
+    - ogg
```

Comment:
Whatever you think is best. 👍

Would add a PR to update the [pinning script]( https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py ) with this and any other pinnings you have been adding/using.